charts/collabora-online/0.0.1/values.yaml

```diff
@@ -0,0 +1,26 @@
+image:
+  repository: collabora/code
+  tag: 6.4.6.2
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 9980
+    nodePort: 30980
+
+env:   
+  domain: nextcloud\.domain\.tld       
+  dictionaries: de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru              
+  username: admin
+  password: changeme
+  extra_params:
```

Comment:
Be carefull about setting default values in values.yaml.
They should always be overridden by questions.yaml, but sometimes people mis values they set before or expect them to exist after questions.yaml

Best practice: use the least possible options in values.yaml